examples/oft-solana/package.json

```diff
@@ -82,7 +82,7 @@
     "ethereumjs-util": "^7.1.5",
     "ethers": "^5.7.2",
     "fp-ts": "^2.16.2",
-    "hardhat": "2.22.3",
+    "hardhat": "^2.22.10",
```

Comment:
Doesn't this break the CLI tasks with the "not iterable" error?

Reply:
@janjakubnanista @ryandgoulding

Reply:
It has been resolved in devtools.  If you are using latest code it is going to work with 2.22.10+